network-api/networkapi/wagtailpages/rss.py

```diff
@@ -37,10 +36,7 @@ def item_link(self, item):
     def item_description(self, item):
         page = item.specific
         html = str(page.body)
-        parsed = BeautifulSoup(html, 'html.parser')
-        text = parsed.get_text()[:1000]
-        if len(text) == 1000:
-            text = f'{text}[...]'
+        text = Truncator(html).chars(1000, html=True)
```

Comment:
On 2nd thought, I should have renamed `text` to something more descriptive, something like `description` 🙈 ?

Reply:
keeping the name is fewer changes =)